tempesta_fw/t/functional/wrk/post_big.lua

```diff
@@ -0,0 +1,885 @@
+wrk.method  = "POST"
+wrk.headers = {["Content-Type"]="text/plain"}
+wrk.body    = [[
+    I
+Мой дядя самых честных правил,
```

Comment:
Unexpected but funny. Usually i saw that some repeated pattern was used for that. It's said that it's more helpful to catch garbaging of recieve-responce buffers. That do you think here?

Reply:
I think, in cases of some memory leaks or smth. else, non-repeating text can be more useful, because it's easier to identify data.